test_rendering/spec/ol/map.test.js

```diff
@@ -16,6 +16,7 @@ describe('ol.rendering.Map', function() {
     target = createMapDiv(50, 50);
 
     var vectorLayer = new ol.layer.Vector({
+      zIndex: 0,
```

Comment:
I dug into this issue a bit more and found the underlying cause.  I'll include the fix in the next commit.